languages/julia/exercises/concept/annalyn2/runtests.jl

```diff
@@ -0,0 +1,137 @@
+using SimpleMock
```

Comment:
I think it might be better if we write a tiny mocking library instead of using SimpleMock. At least locally, SimpleMock is rather slow when it fails and would easily hit the 10s time limit. It also adds unneeded complexity to require the student to download dependencies.